parser/sql/dialect/doris/src/main/antlr4/imports/doris/DDLStatement.g4

```diff
@@ -359,6 +359,43 @@ createView
       (WITH (CASCADED | LOCAL)? CHECK OPTION)?
     ;
 
+// DORIS ADDED BEGIN
+createMaterializedView
+    : CREATE MATERIALIZED VIEW ifNotExists? name
+    (LP_ columnDefinition RP_)? buildMode?
+    (REFRESH refreshMethod? refreshTrigger?)?
+    ((DUPLICATE)? KEY keys= LP_ identifierList RP_)?
+    commentClause? partitionClause? distributedbyClause? propertiesClause?
+    AS select
+    ;
+// DORIS ADDED END
+
+// DORIS ADDED BEGIN
+buildMode
+    : BUILD (IMMEDIATE | DEFERRED)
+    ;
+// DORIS ADDED END
+
+// DORIS ADDED BEGIN
+refreshMethod
+    : COMPLETE | AUTO
+    ;
+// DORIS ADDED END
+
+// DORIS ADDED BEGIN
+refreshTrigger
+    : ON MANUAL
```

Comment:
Please keep the systax in one line unless it contains more than 5 branches. You can read SQL Parser of Conduct - https://shardingsphere.apache.org/community/en/involved/conduct/code/#sql-parser-of-conduct